docs/components/modeler/web-modeler/new-context-pad.md

```diff
@@ -0,0 +1,36 @@
+---
+id: new-context-pad
+title: New context pad
+description: The new context pad is a feature that allows you to easily add elements to your diagram and to perform actions on elements.
+---
+
+<span class="badge badge--cloud">Camunda 8 only</span>
+<span class="badge badge--beta">Alpha</span>
```

Comment:
Will adjust this in a separate PR 👍

Reply:
Can add an alpha badge here now, as added in https://github.com/camunda/camunda-docs/pull/3357/files.